### ISSUE TITLE
Respond by email failing

### ISSUE THREAD

**Issue by tankerkiller125** [julianlam/nodebb-plugin-emailer-sendgrid#173]
I'm seeing the following error with the latest version of this plugin and NodeBB v3.1.4

```
2023-06-11T15:47:46.259Z [4568/1167363] - error: POST /plugins/emailer-sendgrid/webhook
TypeError: Plugins.fireHook is not a function
    at Emailer.receive (/root/sysadminszone/nodebb/node_modules/nodebb-plugin-emailer-sendgrid/index.js:76:10)
    at Layer.handle [as handle_request] (/root/sysadminszone/nodebb/node_modules/express/lib/router/layer.js:95:5)
    at next (/root/sysadminszone/nodebb/node_modules/express/lib/router/route.js:144:13)
    at Form.<anonymous> (/root/sysadminszone/nodebb/node_modules/connect-multiparty/index.js:114:7)
    at Form.emit (node:events:525:35)
    at Form.emit (node:domain:489:12)
    at /root/sysadminszone/nodebb/node_modules/multiparty/index.js:612:12
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
```

I have rebuild and restarted the NodeBB server multiple times and I'm still getting this error.

**Comment by julianlam** [comment]
Thanks for reporting, will fix

**Comment by barisusakli** [comment]
@tankerkiller125 thanks for reporting, fixed in 3.1.1
